LEAF_Request_Portal/form.php

```diff
@@ -3048,14 +3053,32 @@ public function query($inQuery)
                         AND deleted = 0', array());
             foreach ($res2 as $item)
             {
-                if($data[$item['recordID']]['recordResolutionData']['fulfillmentTime'] == null
-                    || $data[$item['recordID']]['recordResolutionData']['fulfillmentTime'] < $item['fulfillmentTime']) {
+                if($data[$item['recordID']]['recordResolutionData']['fulfillmentTime'] == null || $data[$item['recordID']]['recordResolutionData']['fulfillmentTime'] < $item['fulfillmentTime']) {
```

Comment:
Need this line broken into two like the one you deleted.  Scrolling off the edge of the page isn't too much of an issue with just html.  But since this is a conditional statement, it needs to be visible without scrolling.

Reply:
this is fixed.